File: 题库标准算法/1.两数之和/C++/Solution1.0.0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target)
    {
        vector<int> result(2);
        const unsigned short length = (unsigned short)nums.size();

        for(unsigned short i = 0; i < length - 1; i++)
        {
            for(unsigned short j = (unsigned short)(i + 1); j < length; j++)
            {
                if(nums[i] + nums[j] == target)
                {
                    result[0] = i;
                    result[1] = j;

                    return result;
                }
            }
        }

        return result;
    }
};

File: 题库标准算法/1.两数之和/Java/Solution1.0.0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int[] result = new int[2];

        for (short i = 0; i < nums.length - 1; i++)
        {
            for (short j = (short) (i + 1); j < nums.length; j++)
            {
                if (nums[i] + nums[j] == target)
                {
                    result[0] = i;
                    result[1] = j;

                    return result;
                }
            }
        }

        return result;
    }
}
